packages/opentelemetry-base/src/ExportResult.ts

```diff
@@ -1,5 +1,5 @@
-/*!
- * Copyright 2019, OpenTelemetry Authors
```

Comment:
Is it really required to update the year on every copyright header?

Reply:
no it was before your comment and I change the eslint config in order to not do that on other plugins

Reply:
it's because of "!" and it replace the comment by the template

Reply:
"!" was required because the old plugin needed it but with eslint it's not needed anymore and we shouldn't have "!" in the header

Reply:
I edited the rule.